src/buildHTML.js

```diff
@@ -1231,6 +1247,19 @@ groupTypes.leftright = function(group, options) {
         ["minner", style.cls()], inner, options);
 };
 
+groupTypes.middle = function(group, options) {
+    var middleDelim;
+    if (group.value.value === ".") {
+        middleDelim = makeNullDelimiter(options, []);
+    } else {
+        middleDelim = delimiter.sizedDelim(
+            group.value.value, 1, options,
+            group.mode, []);
```

Comment:
There's another comment on this. If LaTeX compatibility were an absolute requirement, then yes, the parser would have to catch every stray `\middle`. But that didn't seem super easy in katex.